lib/std/os/linux.zig

```diff
@@ -1005,7 +1005,7 @@ pub fn recvfrom(fd: i32, noalias buf: [*]u8, len: usize, flags: u32, noalias add
     return syscall6(.recvfrom, @bitCast(usize, @as(isize, fd)), @ptrToInt(buf), len, flags, @ptrToInt(addr), @ptrToInt(alen));
 }
 
-pub fn shutdown(fd: i32, how: i32) usize {
+pub fn shutdown(fd: i32, how: c_int) usize {
```

Comment:
Native zig types are preferred, why this change?

Reply:
ok I've removed this change and modified the new declaration in `c.zig` to use `i32`.